package.json

```diff
@@ -17,9 +17,8 @@
     "start:docker-init": "docker-compose up --build",
```

Comment:
I think we should drop these scripts, and just rely on the standard `docker-compose` commands. The root README already says to use `docker-compose up`